### PR TITLE
Fx improvements

### PR DESCRIPTION
**Pull request by nadult**
- stacking debuffs
- added glow to fire (looks much better); glow is drawn behind objects

### CODE REVIEW

File: event_listener.h [hunk 1]
```diff
@@ -98,6 +98,7 @@ namespace EventInfo {
     FXName effect;
     Color color = Color::WHITE;
     Vec2 targetOffset = Vec2(0, 0);
+    bool alwaysShow = false;
```
Comment (miki151) [inline event_listener.h]: What is this for? To let the player always see where the enemy escaped to?

Comment (nadult) [reply event_listener.h]: No, this is for TELEPORT_IN: when player teleports outside of current view, fx will be invisible.

Comment (miki151) [reply event_listener.h]: But you set it to true for targetPos, which is where the creature is moved. Note that this should work when both player and enemies teleport.

Perhaps the right way to do this is:
addEvent [ TELEPORT_OUT ]
moveCreature
addEvent [ TELEPORT_IN ]

(without this extra flag)

Comment (nadult) [reply event_listener.h]: OK, if moveCreature will update visiblityMap immediately then this will work without alwaysShow flag.

**Pull request merged**